src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -107,6 +107,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
     request.setAttribute("conversation", conversation);
     request.setAttribute("messages", messages);
+
+    String addNewUserParameter = request.getParameter("add_new_user");
```

Comment:
Maybe rename this to something else since you already have an `addNewUser` parameter in your post handler. May get confusing